Formula/hyperkit.rb

```diff
@@ -0,0 +1,49 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20180123",
+    :revision => "8bbe1da0166553a5b0ccff170bf9ccbd7d10753e"
```

Comment:
Can we build from a source archive rather than git checkout?

Reply:
Looks like we could build from the source archives at https://github.com/moby/hyperkit/releases